src/renderers/shaders/ShaderChunk/transmission_pars_fragment.glsl.js

```diff
@@ -184,12 +184,15 @@ export default /* glsl */`
 		// Sample framebuffer to get pixel the refracted ray hits.
 		vec4 transmittedLight = getTransmissionSample( refractionCoords, roughness, ior );
 
-		vec3 attenuatedColor = applyVolumeAttenuation( transmittedLight.rgb, length( transmissionRay ), attenuationColor, attenuationDistance );
+		vec3 transmittance = diffuseColor * volumeAttenuation( length( transmissionRay ), attenuationColor, attenuationDistance );
+		vec3 attenuatedColor = transmittance * mix( vec3( 0.5 ), transmittedLight.rgb, transmittedLight.a );
```

Comment:
So here I'm mixing in a neutral amount of light (0.5) to represent the unknown light that will come in from the background color.